nextcord/client.py

```diff
@@ -2869,7 +3001,38 @@ class be provided, it should be a subclass of :py:class:`Interaction`.
         .. note::
 
             This is synchronous due to how slash commands are implemented.
-        """
+
+        Warns
+        --------
+        ~FutureWarning
```

Comment:
I don't think it's worth documenting this warning, considering that other functions throughout the library do not either.